docs/ftldns/configfile.md

```diff
@@ -263,7 +272,7 @@ Controls if *FTL*DNS should print extended details about regex matching into `pi
 
 **[More details](../regex/overview.md)**
 
-#### `DEBUG_API=false|true` {#debug_api data-toc-label='Telnet API'}
+#### `DEBUG_API=false|true` {#debug_api data-toc-label='Telnet'}
```

Comment:
```suggestion
#### `DEBUG_API=false|true` {#debug_api data-toc-label='API'}
```

Reply:
I removed `API` on purpose - with this auto-hint thing it completely destroys the TOC entry and the heading. Have a look at the current version: https://docs.pi-hole.net/ftldns/configfile/#debug_apifalsetrue-debug_api-data-toc-labeltelnet-api

Reply:
Okay, so we need to change this for v6.0 but this is okay as a workaround for now.